eureka-client/build.gradle

```diff
@@ -37,4 +37,5 @@ dependencies {
     testCompile "org.mockito:mockito-core:${mockitoVersion}"
     testCompile "org.mock-server:mockserver-netty:${mockserverVersion}"
     testCompile "com.netflix.governator:governator:${governatorVersion}"
+    testCompile 'org.powermock:powermock-api-mockito:1.6.6'
```

Comment:
done

Reply:
OK, that looks better because it matches the approach in the project but what I was really getting at was whether we could run a version which is compatible with the project rather than the Diffblue-preferred version.

Reply:
I tried it with the same version but it didn't work :(